config/webpacker.yml

```diff
@@ -0,0 +1,63 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  source_path: app/assets/javascripts
+  source_entry_path: packs
+  public_root_path: public
+  public_output_path: packs
+  cache_path: tmp/cache/webpacker
+  webpack_compile_output: true
+
+  # Additional paths webpack should lookup modules
+  # ['app/assets', 'engine/foo/app/assets']
+  additional_paths: []
```

Comment:
I think this can be used to replace this trick I used to load yarn stuff from node_modules into the bundle.
See config/initializers/assets.rb, we have:
Rails.application.config.assets.paths << Rails.root.join('node_modules')

Reply:
angular-google-maps.min.js for example is loaded this way right now. Something to test as we start using webpacker.

Reply:
Yes, I think definitively, using `webpacker` to prepare our assets could be a great idea (even without talking about tailwind). 
Because seeing, this kind of line is really ... ouch !
```
Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Reply:
ok 👍